issues: fix issue creation

tbg opened the pull request:
We were specifying `C-test-failure` twice. Github actually doesn't care,
but let's fix it anyway.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/44393)
<!-- Reviewable:end -->

andreimatei reviewed (commented):
LGTM

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/44393#-:-Lzb_tLYE49jskWW1tz0:bb6ywmx)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @andreimatei)



<!-- Sent from Reviewable.io -->

tbg committed:
issues: fix issue creation

We were specifying `C-test-failure` twice. Github actually doesn't care,
but let's fix it anyway.

Release note: None

tbg commented:
bors r=andreimatei

TFTR!
